src/Document/Areabrick/Drop.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace App\Document\Areabrick;
+
+use App\Document\Areabrick\AbstractAreabrick;
+
+class Drop extends AbstractAreabrick
+{
+    public function getName()
```

Comment:
Please add phpdoc with return types

Reply:
Also, if it possible, please use translate for name, like `areabrick.drop.name` and add it to translations

Reply:
Done